package.json

```diff
@@ -14,7 +14,7 @@
     "permissionless": "^0.1.36",
     "react": "^18",
     "react-dom": "^18",
-    "viem": "^2.17.3"
+    "viem": "^2.18.8"
```

Comment:
important: we needed to bump version to have access to general `sign` function on viem accounts